rollup.config.js

```diff
@@ -119,26 +120,30 @@ export default [
   /** ****************************
    * CJS Module Bundling
    */
-  {
-    input: 'src/lib/widget/inline.svelte',
-    onwarn: warningHandler,
-    output: {
-      file: path.join(packageFolder, 'inline.cjs'),
-      format: 'cjs',
-      sourcemap: true,
-      inlineDynamicImports: true,
-    },
-    plugins,
-  },
-  {
-    input: 'src/lib/widget/modal.svelte',
-    onwarn: warningHandler,
-    output: {
-      file: path.join(packageFolder, 'modal.cjs'),
-      format: 'cjs',
-      sourcemap: true,
-      inlineDynamicImports: true,
-    },
-    plugins,
-  },
-];
+  CI
+    ? {
+        input: 'src/lib/widget/inline.svelte',
+        onwarn: warningHandler,
+        output: {
+          file: path.join(packageFolder, 'inline.cjs'),
+          format: 'cjs',
+          sourcemap: true,
+          inlineDynamicImports: true,
+        },
+        plugins,
+      }
+    : undefined,
+  CI
+    ? {
+        input: 'src/lib/widget/modal.svelte',
+        onwarn: warningHandler,
+        output: {
+          file: path.join(packageFolder, 'modal.cjs'),
+          format: 'cjs',
+          sourcemap: true,
+          inlineDynamicImports: true,
+        },
+        plugins,
+      }
+    : undefined,
+].filter(Boolean);
```

Comment:
This syntax always trips me up when I see it. I'm not a huge fan of this pattern, but I think it's fine for this context.